src/Microsoft.VisualStudio.Editors/PropPages/Strings.resx

```diff
@@ -131,4 +131,15 @@
     <value>http://go.microsoft.com/fwlink/?LinkID=287120</value>
     <comment>{Locked}</comment>
   </data>
+  <data name="LanguageVersionAutomaticallySelected" xml:space="preserve">
+    <value>Automatically selected based on framework version</value>
+    <comment>Refers to the C# language version being automatically selected based on the target framework</comment>
+  </data>
+  <data name="CantSelectLanguageVersionLinkLabel" xml:space="preserve">
+    <value>Why can't I select a different C# version?</value>
+  </data>
+  <data name="CantSelectLanguageVersionFWLink" xml:space="preserve">
+    <value>https://aka.ms/csharp-versions</value>
+    <comment>{Locked}</comment>
```

Comment:
Is this a special syntax for prevening a string from being translated? If so is it understood by human or machine?

Reply:
Yes, it's one of a set of specialized comments for exercising some control over the translation process. This one means that the string must preserve the original value across all locales. Our human translators understand these, and they are "machine-readable" to a certain extent. This one, for example, will prevent the string from actually being placed in the .xlf files--there's no point in the translators even seeing it if we don't want it translated.

Reply:
Nice. Are these documented somewhere?

Reply:
There's an internal document. I'll send around the link.